stackslib/src/chainstate/stacks/boot/signers_voting_tests.rs

```diff
@@ -258,7 +259,8 @@ fn vote_for_aggregate_public_key_in_last_block() {
     );
 
     let cycle_id: u128 = current_reward_cycle;
-    let aggregated_public_key: Point = Point::new();
+    let aggregate_public_key: Point = Point::new();
+    let aggregate_public_key_1: Point = Point::G();
```

Comment:
Is there a better way to create aggregate public keys for testing ? @MarvinJanssen

Reply:
You could do:

```rust
let private_key = StacksPrivateKey::from_seed(&[3]);
let public_key = StacksPublicKey::from_private(&private_key);
```

And change the seed input to different values.

If you want to generate actual aggregate public keys then take a look at `TestSigners`.

```rust
let test_signers = TestSigners::default();
let public_key = test_signers.aggregate_public_key.clone();
```

Reply:
Thank you. Code has been updated